src/api/sources/sds.ts

```diff
@@ -12,41 +14,32 @@ interface SDSInstance {
   };
 }
 
-interface SDSResult {
-  acceptedName: string | null;
-  commonName: string | null;
-  instances: SDSInstance[];
-  scientificName: string;
-  status: string[];
-}
-
-async function get(
-  scientificName: string,
-  latitude?: number,
-  longitude?: number,
-  date?: string,
-): Promise<SDSResult> {
-  let URL = `${import.meta.env.VITE_API_ALA}/sds-webapp/ws/${scientificName}`;
+async function getInstances(guid: string, state: string) {
+  const URL = `${
+    import.meta.env.VITE_API_ALA
+  }/sensitive/api/report?taxonId=${guid}&stateProvince=${state}&country=AUS`;
+  const { sensitive, valid, report } = await (await fetch(URL)).json();
 
-  // Append optional lat/lng parameters
-  if (latitude !== undefined && longitude !== undefined)
-    URL += `/location/${latitude}/${longitude}`;
+  return valid && sensitive ? report.taxon.instances : [];
+}
 
-  // Append optional date parameters
-  if (date !== undefined) URL += `/date/${date}`;
+async function get(guid?: string): Promise<SDSInstance[]> {
+  if (!guid) return [];
 
   // Wrap in a try-catch to handle
   try {
-    const data = await (await fetch(URL)).json();
-    if (!data.instances) throw new Error('Invalid request');
-    return data;
+    return (
+      await Promise.all(
+        ['QLD', 'NT', 'NSW', 'WA', 'SA', 'ACT'].map((state) => getInstances(guid, state)),
```

Comment:
Just wondering why the list doesn't include VIC or TAS?

Reply:
Good catch! I was testing with only a subset & forgot to add the rest. I've just fixed this then 👍